cowbird/api/webhooks/views.py

```diff
@@ -102,8 +102,8 @@ def post_permission_webhook_view(request):
                     is_in=True,
                     http_error=HTTPBadRequest,
                     msg_on_fail=s.PermissionWebhook_POST_BadRequestResponseSchema.description)
-    service_name = ar.get_multiformat_body(request, "service_name")
-    resource_id = ar.get_multiformat_body(request, "resource_id")
+    service_name = ar.get_multiformat_body_raw(request, "service_name")
```

Comment:
I would get an error using `get_multiformat_body` because the function checks that the value is `not_none`, but Magpie sends a `None` value for the service name, unless the permission changed is applied directly to a service-type resource.

Reply:
Tried it, but the `verify_param` function doesn't seem to support using a tuple of different types when using the `is_type` comparison.
The function could probably be adjusted to add this support, but since it doesn't seem to be a priority, I'll keep the raw format for now.

Reply:
Does it fail in the pre-parameter checks? 
The check itself should support a tuple here: 
https://github.com/Ouranosinc/cowbird/blob/638ac61a45ecfa864ae8b78aabd022434bc5b79b/cowbird/api/exception.py#L226-L228

Reply:
Ok. I checked and indeed caused by the following pre-check that fails the following condition:
https://github.com/Ouranosinc/cowbird/blob/638ac61a45ecfa864ae8b78aabd022434bc5b79b/cowbird/api/exception.py#L158
https://github.com/Ouranosinc/cowbird/blob/638ac61a45ecfa864ae8b78aabd022434bc5b79b/cowbird/api/exception.py#L165-L167

This should be fixed by adjusting like so:
```python
is_cmp_typ = isinstance(param_compare, type) or (
    hasattr(param_compare, "__iter__") and 
    all(isinstance(cmp, type) for cmp in param_compare)
)
```